extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -261,7 +261,6 @@ public function getWikiVariables() {
 				'enableDiscussions' => $wgEnableDiscussions,
 				'enableDiscussionsImageUpload' => $wgEnableDiscussionsImageUpload,
 				'enableFandomAppSmartBanner' => !empty( $enableFAsmartBannerCommunity ),
-				'enableFeedsAndPosts' => $wgEnableFeedsAndPostsExt,
```

Comment:
why do we remove it?

Reply:
bc this endpoint is not used by any of ember apps